roles/zabbix_proxy/tasks/Debian.yml

```diff
@@ -2,45 +2,10 @@
 - name: "Debian | Set short version name"
   ansible.builtin.set_fact:
     zabbix_short_version: "{{ zabbix_proxy_version | regex_replace('\\.', '') }}"
-    zabbix_python_prefix: "python{% if ansible_python_version is version('3', '>=') %}3{% endif %}"
     zabbix_underscore_version: "{{ zabbix_proxy_version | regex_replace('\\.', '_') }}"
   tags:
     - always
 
-- name: "Debian | Installing lsb-release"
```

Comment:
I thought I remember seeing a note somewhere about you putting lsb-release back into it to support Raspbian, otherwise that would be a breaking change.

Reply:
Yes, the lsb-release comes pre-installed on all (or atleast the latest bullseye and bookworm) raspbian systems, and ansible_facts populates it on those systems. You'll see in `zabbix_repo_deb_url:`  the construction `ansible_facts.lsb.id | default(ansible_facts['distribution']) | lower`. We try using `lsb.id`, and if it's not populated we fall back to `distribution`. With the fallback, we don't need to install the package on debian/ubuntu. Details in a7e07514cc227dbd6d56959ab9db6fa2c07d27ff

Reply:
Yeah, so the reason they wanted it installed was because if you try to access ansible_facts.lsb.id without it being there, you'll get an error.

**Unless** you catch that error with a `| default(alternative_variable`, which is what we're doing here. Thereby saving us the trouble of having to install this package on systems that don't require it, just so they too populate a value to `lsb.id` that we don't need.

This is not a breaking change.

Reply:
`zabbix_repo_deb_url: "http://repo.zabbix.com/zabbix/{{ zabbix_proxy_version }}/{{ ansible_facts.lsb.id | default(ansible_facts['distribution']) | lower }}{% if ansible_facts['architecture'] == 'aarch64' and ansible_facts.lsb.id | default(ansible_facts['distribution']) == 'Debian' %}-arm64{% endif %}"`. 

It defaults to distribution which will show Debian and no Raspbian which is why it was put in there in the first place.  If you go back through the conversation of https://github.com/ansible-collections/community.zabbix/pull/1145, I wasn't a fan of adding Rasbian in there in the first place but the decision was made, so it was added (albeit as "best effort").  That being said, because it was added, to remove the capability (installing this on a system that doesn't properly identify itself as Raspbian) is a breaking change.

That said, given the fact that you said elsewhere to just wait (trust me, I have a few PRs ready to go too) not worried about this.

Reply:
I'm not sure if I misunderstand you, so apologies for being very verbose here. I've gathered and included a dump of `architecture`, `lsb` and rendered `zabbix_repo_deb_url` for 3 raspbian systems, debian-aarch64 and the 3 debian and 3 docker containers currently in use in testing.

The main point I want to bring focus to is that the debian-docker images do not supply the `lsb` section. And if you try to read `lsb.id` when it's not populated, you get the `'dict object' has no attribute 'id'.`, **but only** for the systems with an empty `lsb`. Which is why the guy in #1145 wanted to install lsb-release to these debian containers. **If** he had used `lsb.id | default(other_thing)` he would not have seen this error on these debian containers.

In short, only the raspbian systems need to use the `lsb.id` and have a value there for us to identify them, which they do. Some of the other systems also populate this section, which is fine and we can use that value. If it is not supplied, we can prevent the `dict object has no attribute id` by defaulting that value to distribution, which is perfectly fine for both debian and ubuntu.

Hopefully this clears things up, I'm sorry I'm not able to explain this better.

Also, I noticed #1219 which will break raspbian systems on aarch64. Apparently ubuntu also has an -arm64 repo now. So I think the condition should change to `lsb.id in ['Debian', 'Ubuntu']`

```
TASK [display arch] ***
ok: [raspbian-bookworm] => {
    "ansible_facts.architecture": "aarch64"
}
ok: [raspbian-bullseye] => {
    "ansible_facts.architecture": "aarch64"
}
ok: [raspbian-buster] => {
    "ansible_facts.architecture": "aarch64"
}
ok: [debian-aarch64] => {
    "ansible_facts.architecture": "aarch64"
}
ok: [docker-ubuntu2204-ansible] => {
    "ansible_facts.architecture": "x86_64"
}
ok: [docker-ubuntu2004-ansible] => {
    "ansible_facts.architecture": "x86_64"
}
ok: [docker-ubuntu1804-ansible] => {
    "ansible_facts.architecture": "x86_64"
}
ok: [docker-debian12-ansible] => {
    "ansible_facts.architecture": "x86_64"
}
ok: [docker-debian11-ansible] => {
    "ansible_facts.architecture": "x86_64"
}
ok: [docker-debian10-ansible] => {
    "ansible_facts.architecture": "x86_64"
}

TASK [display ansible_facts.lsb] ***
ok: [raspbian-bookworm] => {
    "ansible_facts.lsb": {
        "codename": "bookworm",
        "description": "Raspbian GNU/Linux 12 (bookworm)",
        "id": "Raspbian",
        "major_release": "12",
        "release": "12"
    }
}
ok: [raspbian-bullseye] => {
    "ansible_facts.lsb": {
        "codename": "bullseye",
        "description": "Raspbian GNU/Linux 11 (bullseye)",
        "id": "Raspbian",
        "major_release": "11",
        "release": "11"
    }
}
ok: [raspbian-buster] => {
    "ansible_facts.lsb": {
        "codename": "buster",
        "description": "Raspbian GNU/Linux 10 (buster)",
        "id": "Raspbian",
        "major_release": "10",
        "release": "10"
    }
}
ok: [debian-aarch64] => {
    "ansible_facts.lsb": {
        "codename": "bookworm",
        "description": "Debian GNU/Linux 12 (bookworm)",
        "id": "Debian",
        "major_release": "12",
        "release": "12"
    }
}
ok: [docker-ubuntu2204-ansible] => {
    "ansible_facts.lsb": {
        "codename": "jammy",
        "description": "Ubuntu 22.04.4 LTS",
        "id": "Ubuntu",
        "major_release": "22",
        "release": "22.04"
    }
}
ok: [docker-ubuntu2004-ansible] => {
    "ansible_facts.lsb": {
        "codename": "focal",
        "description": "Ubuntu 20.04.6 LTS",
        "id": "Ubuntu",
        "major_release": "20",
        "release": "20.04"
    }
}
ok: [docker-ubuntu1804-ansible] => {
    "ansible_facts.lsb": {
        "codename": "bionic",
        "description": "Ubuntu 18.04.6 LTS",
        "id": "Ubuntu",
        "major_release": "18",
        "release": "18.04"
    }
}
ok: [docker-debian12-ansible] => {
    "ansible_facts.lsb": {}
}
ok: [docker-debian11-ansible] => {
    "ansible_facts.lsb": {}
}
ok: [docker-debian10-ansible] => {
    "ansible_facts.lsb": {}
}

TASK [display zabbix_repo_deb_url] ***
ok: [raspbian-bullseye] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/raspbian"
}
ok: [raspbian-buster] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/raspbian"
}
ok: [raspbian-bookworm] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/raspbian"
}
ok: [docker-ubuntu2204-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/ubuntu"
}
ok: [docker-ubuntu2004-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/ubuntu"
}
ok: [debian-aarch64] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/debian-arm64"
}
ok: [docker-debian12-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/debian"
}
ok: [docker-debian11-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/debian"
}
ok: [docker-ubuntu1804-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/ubuntu"
}
ok: [docker-debian10-ansible] => {
    "zabbix_repo_deb_url": "http://repo.zabbix.com/zabbix/6.0/debian"
}
```

Reply:
OK, you've sold me.  When we were looking at https://github.com/ansible-collections/community.zabbix/pull/1145 I didn't realize it was simply that it was lacking in the container image, I thought it was lacking in the actual image as well by default.  If I had realized that, I would have just said to install it as part of the molecule prep playbook.  To be completely honest I wasn't even familiar with the LSB stuff until it popped up, and only have a passing familiarity with it now.